src/PrestaShopBundle/Resources/config/routing/admin/improve/international/currencies.yml

```
@@ -80,3 +80,21 @@ admin_currencies_get_reference_data:
     _controller: 'PrestaShopBundle:Admin\Improve\International\Currency:getReferenceData'
     _legacy_controller: AdminCurrencies
     _legacy_link: AdminCurrencies:getReferenceData
+
+admin_currencies_bulk_delete:
+  path: /bulk-delete
+  methods: [POST]
```

```suggestion
  methods: [DELETE]
```
or POST and DELETE for the moment? :thinking:

---

It's a BC to remove method ;)

---

It's a new path, can't be a BC break :thinking:

---

Oops for me, in this case.